marc_to_solr/lib/traject_config.rb

```diff
@@ -1352,7 +1352,9 @@
   fields = []
   dates = []
   embargoes = []
-  # dont check for scsb
+  # Don't index if 951$a is 'Not Available' - inactive portfolio
+  # Don't check for scsb
+  next if inactive_electronic_portfolio(record)
```

Comment:
Is there ever a time when one of the 951s is inactive, but other ones aren't? If so, I wonder if this should go down in 1359 similar to `next unless alma_951(record)`?

Reply:
yes there is this possibility. let me check

Reply:
I'll try to find a fixture with active and inactive